pkg/cmd/util/output/backup_storage_location_printer.go

```diff
@@ -50,6 +51,11 @@ func printBackupStorageLocation(location *velerov1api.BackupStorageLocation) []m
 		Object: runtime.RawExtension{Object: location},
 	}
 
+	isDefault := ""
+	if location.Spec.Default {
+		isDefault = "true"
```

Comment:
Please make this uppercase. It will match the value of columns like PHASE [Available/Unavailable].

Reply:
The Default is a _boolean_ type, so calling `kubectl get backupstoragelocations` prints `true`.
Therefore, I make the velero `backup-location get` output format aligns with `kubectl get backupstoragelocations`.

If we want to match the value of columns like PHASE, then we have to change the data type to _string_, it makes the comparison and the meaning more complicated, WDYT?

Reply:
Right, this makes sense. Good call.